packages/svelte/src/compiler/phases/3-transform/client/transform-client.js

```diff
@@ -473,10 +473,12 @@ export function client_component(source, analysis, options) {
 	if (options.dev) {
 		if (options.filename) {
 			let filename = options.filename;
-			if (/(\/|\w:)/.test(options.filename)) {
```

Comment:
Should we keep the old code around in an else-if block?

Reply:
in any case we only use `filename` in the dev build IIRC, so it's probably fine anyway (though we should maybe make it so that `filename` is inaccessible to the compiler output, rather than merely unaccessed)

Reply:
confirmed that we don't use `filename` in compiler output in prod, and have added a note to the JSDoc annotation — hopefully that will be sufficient to discourage any unsafe usage